src/jio-report-a-problem.ts

```diff
@@ -45,6 +45,12 @@ export class ReportAProblem extends LitElement {
   @property()
   githubBranch = 'master';
 
+  /**
+   * Whether to use the plugin site report URL
+   */
+  @property({ type: Boolean })
+  usePluginSiteReportUrl = false;
```

Comment:
lets change this to a string
```
@property({ type: String })
template = ""
```

Also add the attribute to jio-footer

```
@property({ type: String })
reportAProblemTemplate = ""
```


Then have jenkins.io provide reportAProblemTemplate="4-bug.yml"

then its opt in for the repos that have templates.